pyroomacoustics/transform/dft.py

```diff
@@ -1,5 +1,5 @@
 # Author: Eric Bezzam
-# Last modification: July 2, 2018
+# Last modification: February 22, 2019
 
 """
 Class for performing the Dicrete Fourier Transform (DFT) and inverse DFT for 
```

Comment:
`Dicrete` -> `Discrete`
(I know this comes from a previous commit, but let's fix it here 😄 )